.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,60 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+    inputs:
+      subsystems:
+        description: Subsystem to run the update for
+        required: false
+        default: '["url"]'
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem: ${{ fromJSON(github.event.inputs.subsystems || '["url"]') }}
+    fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v4.1.7
+        with:
+          persist-credentials: false
+
+      - name: Update WPT for subsystem ${{ matrix.subsystem }}
+        run: |
+          git node wpt ${{ matrix.subsystem }}
```

Comment:
this will not work until node-core-utils is [installed](https://github.com/nodejs/node-core-utils#install).